### PR TITLE
bpo-38323: Skip SubprocessMultiLoopWatcherTest as they can hang the test suite

### PR DESCRIPTION
**Pull request by pablogsal**
https://bugs.python.org/issue38323

### CODE REVIEW

**Comment by pablogsal** [comment]
@cjerdonek could you take a look?

**Review by cjerdonek — APPROVED** [review]
Yeah, that looks fine.

File: Lib/test/test_asyncio/test_subprocess.py [hunk 1]
```diff
@@ -665,6 +665,8 @@ class SubprocessThreadedWatcherTests(SubprocessWatcherMixin,
 
         Watcher = unix_events.ThreadedChildWatcher
 
+    @unittest.skip("SubprocessMultiLoopWatcher has a race condition \
```
Comment (cjerdonek) [inline Lib/test/test_asyncio/test_subprocess.py]: Actually, it's called `MultiLoopChildWatcher`. Do you want to add the bug number, too?

Comment (pablogsal) [reply Lib/test/test_asyncio/test_subprocess.py]: Yeah, makes sense. Thanks for the catch!

**Comment by vstinner** [comment]
>      @unittest.skip("bpo-38323: MultiLoopChildWatcher has a race condition 

FYI if a skip message contains "bpo-xxx", a buildbot fails when a PR is merged and the buildbot logs contain the skip message, the PR will be attached to bpo-xxx.

IMO the bot attaching PRs to bpo issues should only look into the PR title, not in PR comments.

**Comment by pablogsal** [comment]
> FYI if a skip message contains "bpo-xxx", a buildbot fails when a PR is merged and the buildbot logs contain the skip message, the PR will be attached to bpo-xxx.

Well, hopefully soon we can merge some form of your PR and then remove that skip :)

**Pull request merged**